provision-base.sh

```diff
@@ -8,6 +8,28 @@ ipv6.dhcp-duid=ll
 ipv6.dhcp-iaid=mac
 EOF
 
+# Create connection files for interfaces
+# All connections will have never-default=True except eth0
+for i in {0..5}; do
+  iface="eth$i"
+  cat << EOF > /etc/NetworkManager/system-connections/$iface
+[connection]
+id=$iface
+uuid=$(uuid)
```

Comment:
The uuid does not wotk, I will need to take a look into this

Reply:
add it to ks then

Reply:
Done